mysql-test/extra/binlog_tests/binlog_drop_tbl_with_comment.test

```diff
@@ -21,8 +36,26 @@ SET binlog_ddl_skip_rewrite = ON;
 CREATE TABLE t2 (a VARCHAR(100));
 DROP TABLE t2 /* comment at end */;
 
+CREATE TABLE t (a INT NOT NULL PRIMARY KEY)
+PARTITION BY KEY (a)
+PARTITIONS 2;
+/* comment at start */ DROP TABLE t /* comment at end */;
+
+CREATE TABLE t1 (a INT NOT NULL PRIMARY KEY)
+PARTITION BY KEY (a)
+PARTITIONS 2;
+
+CREATE TABLE t2 (a INT NOT NULL PRIMARY KEY)
+PARTITION BY KEY (a)
+PARTITIONS 2;
+
+--error ER_DROP_MULTI_TABLE
+/* comment at start */ DROP TABLE t1 /*!80017 ,t2*/ /* comment at end */;
```

Comment:
Can you also add a scenario to test a single table DROP TABLE query with versioned comment?

`/* comment at start */ DROP TABLE t1 /*!80017 */ /* comment at end */;`